### PR TITLE
Arm action server and integration with main_engine

### PR DESCRIPTION
**Pull request by paulvazbad**
- [x] Created the arm package
- [ ]  Added action server to the arm package 
- [ ] Define and implement goal/result messages 
- [ ] Dummy pick_up and put_down in the action server
- [ ] Connected action server to the main engine

### CODE REVIEW

File: catkin_home/src/action_selectors/scripts/parser.py [hunk 1]
```diff
@@ -1,114 +1,174 @@
...
+class Parser(object):
+    def __init__(self, pub, pub_resp, pub_story, filename):
+        self.debug_option = True
+        # Im sorry but i gues this is b etter than creating the publisher in each callback.
+        self.pub = pub
+        self.pub_resp = pub_resp
+        self.pub_story = pub_story
+        self.possible_actions = self.loadActions(filename)
+
+    def loadActions(self, filename):
```
Comment (AuroTB) [inline catkin_home/src/action_selectors/scripts/parser.py:36]: Correct this function's indentation.

File: catkin_home/src/action_selectors/scripts/parser.py [hunk 2]
```diff
@@ -1,114 +1,174 @@
...
+
+
+def main():
+
+    # In ROS, nodes are uniquely named. If two nodes with the same
+    # name are launched, the previous one is kicked off. The
+    # anonymous=True flag means that rospy will choose a unique
+    # name for our 'listener' node so that multiple listeners can
+    # run simultaneously.
+
```
Comment (AuroTB) [inline catkin_home/src/action_selectors/scripts/parser.py:160]: maybe delete some of these spaces

File: catkin_home/src/devices/scripts/exampleShow.py [hunk 3]
```diff
@@ -8,9 +8,10 @@
 
 
 def callback(data):
+    print("yeet")
```
Comment (AuroTB) [inline catkin_home/src/devices/scripts/exampleShow.py:11]: Put a more descriptive debugging message, please.

File: catkin_home/src/devices/scripts/exampleShow.py [hunk 4]
```diff
@@ -8,9 +8,10 @@
...
     bridge = CvBridge()
-    cv_image = bridge.imgmsg_to_cv2(data, desired_encoding="passthrough")
-    cv2.imshow("frames",cv_image)
+    cv_image = bridge.imgmsg_to_cv2(data, desired_encoding="bgr8")
+    cv2.imshow("image",cv_image)
     if cv2.waitKey(1) & 0xFF == ord('q'):
```
Comment (AuroTB) [inline catkin_home/src/devices/scripts/exampleShow.py:15]: Question
The topic you are subscribed to called "frames" refers to which of the cameras' topics? Or does it refer to something else?

File: catkin_home/src/main_engine/launch/engine_and_parser.launch [hunk 5]
```diff
@@ -0,0 +1,4 @@
+<launch>
```
Comment (AuroTB) [inline catkin_home/src/main_engine/launch/engine_and_parser.launch:1]: Indentation

File: catkin_home/src/main_engine/launch/main_engine.launch [hunk 6]
```diff
@@ -1,4 +1,3 @@
-<launch>
-<node name="engine" pkg="main_engine" type="engine.py" respawn="true"/>
-<node name="mock" pkg="main_engine" type="mock_command_creator.py" respawn="true"/>
+<launch>
```
Comment (AuroTB) [inline catkin_home/src/main_engine/launch/main_engine.launch:1]: Indentation

File: catkin_home/src/devices/scripts/exampleShow.py (hunk 3 above)
Comment (paulvazbad) [reply catkin_home/src/devices/scripts/exampleShow.py]: no

File: catkin_home/src/devices/scripts/exampleShow.py (hunk 4 above)
Comment (paulvazbad) [reply catkin_home/src/devices/scripts/exampleShow.py]: That topic is published in imageCapture.py and it is possible to change the camera it is reading

**Pull request merged**